### PR TITLE
[CVP-1588] Set default baseDomain variable inside for ipi stage in step-registry

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject, 
These changes help up overriding the baseDomain variable inside the config template of ipi stage.

### CODE REVIEW

**Comment by samvarankashyap** [comment]
@dirgim PTAL

File: ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml [hunk 1]
```diff
@@ -18,5 +18,9 @@ ref:
       default: "2"
       documentation: |-
         The number of AZs to present in the cluster. Accepted values are 1 and 2.
+    - name: BASE_DOMAIN
+      default: "origin-ci-int-aws.dev.rhcloud.com"
+      documentation: |-
+        Base domain is used for setting baseDomain variable in Config template used in ipi-conf-aws-commands.sh
```
Comment (dirgim) [inline ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml:24]: Maybe changing this to something like ```A fully-qualified domain or subdomain name. The base domain of the cloud provider is used when setting up the install configuration of the cluster.```

Comment (samvarankashyap) [reply ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml]: Done

**Comment by wking** [comment]
/retitle [CVP-1588] Set default baseDomain variable inside for ipi stage in step-registry

**Comment by wking** [comment]
Looks reasonable to me, but I'd like to see at least a few green rehearsals.

**Comment by wking** [comment]
Has some green now.

/lgtm

**Pull request merged**